tests/containers/podman_pods.pm

```diff
@@ -18,6 +18,43 @@ use serial_terminal 'select_serial_terminal';
 use version_utils qw(is_sle is_opensuse is_sle_micro);
 use containers::k8s qw(install_k3s uninstall_k3s);
 
+sub check_container_nspid {
+    # expect set hostpid
+    my $hostpid = shift;
+    my $config = 'hello-kubic.yaml';
+    my @errors = ();
+
+    record_info('NSPid', sprintf('Expect HostPid == %s', !!$hostpid ? 'True' : 'False'));
+
+    if ($hostpid) {
+        assert_script_run(qq[sed -i '/containers:/i\\      hostPID: true' $config]);
+    }
+
+    assert_script_run("podman play kube --replace $config");
+    my $pod = script_output(q[podman pod ps --format '{{ .Name }}']);
+    my $container = script_output(q[podman container ps --filter name=.*pod.* --format '{{ .Names }}']);
+    my $pid_on_host = script_output(qq[podman container inspect $container --format '{{ .State.Pid }}']);
+    my $pid_mode = script_output(qq[podman inspect $container --format '{{ .HostConfig.PidMode }}']);
+
+    my @pids = split(/\s+/, script_output("grep NSpid: /proc/$pid_on_host/status"));
+    shift @pids;
+    record_info('Detected', sprintf("%s", join(", ", @pids)));
+
+    if ($hostpid && $pid_mode ne 'host') {
+        push @errors, "HostConfig.PidMode should be set to 'host', currently set to $pid_mode";
```

Comment:
The wording makes it look like the user has to set `HostConfig.PidMode` to `host` and that it has somehow been set to `private`. Since these attributes are implicitly set by Podman and the user can only set `hostPId: true`, is it possible to rephrase this?